x-pack/test/security_solution_cypress/cypress/tasks/entity_analytics.ts

```diff
@@ -42,6 +42,7 @@ export const navigateToNextPage = () => {
 export const riskEngineStatusChange = () => {
   cy.get(RISK_SCORE_SWITCH).should('not.have.attr', 'disabled');
   cy.get(RISK_SCORE_SWITCH).click();
+  cy.get(RISK_SCORE_STATUS).should('have.text', 'On');
```

Comment:
Wait for the enable risk engine response to arrive and make sure it is enabled before proceeding. It fixed the test flakiness 🥳

Reply:
Nice! I was looking at something similar for one of our other flaky tests I wonder if this is the fix